screens/Home/ProductDetailsScreen.tsx

```diff
@@ -1,124 +1,142 @@
-import { StyleSheet, Text, View, Button, Image, TouchableOpacity } from "react-native";
-import React from "react";
+import {
+  StyleSheet,
+  Text,
+  View,
+  ScrollView,
+  Button,
+  Image,
+  TouchableOpacity,
+} from "react-native";
+import React, { useState } from "react";
 import { useNavigation, useRoute } from "@react-navigation/core";
 import { useProductQuery } from "../../services/productsApi";
-import { RouteProp } from "@react-navigation/native"
-type ParamList = { Params: { id: string }; };
+import { RouteProp } from "@react-navigation/native";
+import QuantitySelector from "../../components/QuantitySelector";
+import { scale } from "react-native-size-matters";
 
+type ParamList = { Params: { id: string } };
+type Props = {
+  onPress: () => void;
+};
 
-const ProductDetailsScreen = () => {
+const ProductDetailsScreen = ({ onPress }: Props) => {
   const navigation = useNavigation();
-  const route = useRoute<RouteProp<ParamList, "Params">>()
-  const { id } = route.params
+  const route = useRoute<RouteProp<ParamList, "Params">>();
+  const { id } = route.params;
   const { data } = useProductQuery(id);
+  const [quantity, setQuantity] = useState<any>(1);
 
+  const onAddToCart = () => { };
```

Comment:
Great work! Seems all good to me. Looking forward to the next part.

Reply:
Thank you Ezer!